minio/api.py

```diff
@@ -365,6 +365,8 @@ def get_bucket_policy(self, bucket_name, prefix=""):
         if not policy_dict:
             return policy.Policy.NONE
 
+        if not policy_dict.get('Statement'):
+            raise ValueError("Policy statement cannot be empty")
```

Comment:
This error message is not right too.  As we received invalid JSON from the server, None value set to policy_dict['Statement'].  You would need to indicate the error something like `None policy statement`